### ISSUE TITLE
Cannot run "tcrmatch" executable from different directory

### ISSUE THREAD

**Issue by danielmarrama** [IEDB/TCRMatch#22]
Running the "tcrmatch" executable from outside the "TCRMatch-0.1.1" directory produces an empty output file. Instead, doing this should either: 1) throw an error that tells the user to cd into TCRMatch-0.1.1 or 2) run correctly independent of the working directory.

**Comment by acrinklaw** [comment]
If we are using a makefile approach it's probably best to use ```make install``` target as well as adding ./configure and make uninstall

**Comment by acrinklaw** [comment]
I think this would go well together with #7, an overhaul of the build + test environment including a more fleshed out makefile

**Comment by danielmarrama** [comment]
Yes! That sounds great. I can take a crack at it if I have time this week.

**Comment by schristley** [comment]
FYI, for VDJServer when running jobs on the supercomputer, tools don't usually get installed in system areas like /usr/local and etc., instead the tools are often put in a "scratch" directory where the job runs. Not to discourage you from using GNU autoconf tools but don't `require` the location for "make install" to be where the file must reside.

To be more concrete, here would be an example where I build and install tcrmatch into a local directory, then copy all the binaries to the "scratch" directory. In reality, the compilation is done once and the binaries are re-used over and over again for many jobs.

```
# configure and make tcrmatch
./configure --prefix=./local
make install

# tar up the local dir binaries
tar zcvf binaries.tgz local

# now setup scratch area
cd $SCRATCH
tar zxvf binaries.tgz

# submit job to run tcrmatch
sbatch tcrmatch_job.sh
```

I don't have any problem with tcrmatch expecting the IEDB file in a data subdirectory, because I can set that up in the scratch directory properly, but if it's requiring the "install" directory that could generate an error.

One way to provide flexibility is have a command line parameter to specify the file. Another idea is to have an environment variable (e.g. `TCRMATCH_DATA`).

**Comment by schristley** [comment]
oops, I missed that there is already a command line parameter to specify the database file, so maybe never mind!

**Comment by acrinklaw** [comment]
@danielmarrama is this issue talking about a situation where the user has already placed the binary where they want and on their PATH? I will have to look into it if that is the case, I think I misunderstood yesterday when looking at this.

And if that is the case, I think fixing it will also be useful for @schristley and other HPC users in general

**Comment by acrinklaw** [comment]
I need to add error check to parse_IEDB_data, maybe generalize to "read_data_file" or something similar. This error occurs because there is no error checking for vector of size 0 when reading in data file

**Comment by danielmarrama** [comment]
I believe Will fixed this with this commit 02b507da65f63ef23e91f6a3cbe2533e913bafc2

**Comment by danielmarrama** [comment]
@acrinklaw
